src/tokens/Assets.tsx

```diff
@@ -414,6 +415,20 @@ function AssetsScreen({ navigation, route }: Props) {
     return items
   }, [t, displayPositions])
 
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      headerRight: () =>
+        getFeatureGate(StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW) ? (
+          <TopBarTextButton
+            testID="Assets/ImportTokenButton"
+            onPress={navigateBack}
+            title={t('importToken')}
+            style={styles.topBarTextItem}
+          />
+        ) : null,
```

Comment:
Here is a neat trick widely used: we can use [`&&` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND) instead of ternary + null. Makes code a bit less noisy. 
```suggestion
        getFeatureGate(StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW) && (
          <TopBarTextButton
            testID="Assets/ImportTokenButton"
            onPress={navigateBack}
            title={t('importToken')}
            style={styles.topBarTextItem}
          />
        ),
```